src/prefect/server/api/logs.py

```diff
@@ -45,3 +48,50 @@ async def read_logs(
         return await models.logs.read_logs(
             session=session, log_filter=logs, offset=offset, limit=limit, sort=sort
         )
+
+
+@router.websocket("/out")
+async def stream_logs_out(
+    websocket: WebSocket,
+) -> None:
+    """Open a WebSocket to stream live logs"""
```

Comment:
This docstring is a little wrong, this isn't opening a websocket, this is _serving_ a websocket.